### DISCUSSION TITLE
nanobind::enum_ does not have a member function called export_values()

### DISCUSSION THREAD

**Discussion by efifogel** [wjakob/nanobind#28] [Ideas]
Hi Wenzel,

This binding generation tool seems to have great potential.
Thank you for developing it and enabling open-source access.

`nanobind::enum_` does not have a member function called `export_values()`.
Without it, it seems that it would be a bit hard to port binding code from pybind11 (or boost.python) and keep existing python code that uses the bindings intact.

Best,
Efi

**Comment by wjakob** [comment]
This feature is now implemented on master. Cheers, Wenzel

**Reply by efifogel** [reply]
works like a charm; thanks
